cmd/upgrade.go

```diff
@@ -21,7 +23,22 @@ var upgradeCmd = &cobra.Command{
 	You can always delete this file.
 	`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("upgrade called")
+		// Enable debug output for upgrade command
+		debug.Verbose = true
+		c := cli.New(Version)
+		ok, err := c.IsUpToDate()
+		BailOnError(err)
+
+		if ok {
+			fmt.Println("Your CLI version is up to date.")
+			return
+		}
+
+		fmt.Printf("Upgrading to version %s\n", c.LatestRelease.Version())
+		err = c.Upgrade()
+		BailOnError(err)
+
+		fmt.Println("Upgrade complete \u2714")
```

Comment:
Awww, I really like the check mark. J/k, this along with the "Upgrading to ..." messaging will be dropped in the next PR.